src/utilities/formInputConfigs.js

```diff
@@ -186,7 +186,7 @@ const FORM_INPUT_CONFIGS = seedObject => ({
     initialValue: '',
     key: 'policyNumberFamily',
     validator: input => input.length >= 0 && input.length < 50,
-    isRequired: false,
+    isRequired: true,
```

Comment:
oh same as in the other PR, isn't this required in the schemaaaaa

Reply:
Yeah, I think this one is OK (required in schema and in input config)?

Reply:
lol oops :)